systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaNodePoolResource.java

```diff
@@ -48,14 +48,15 @@ public void create(KafkaNodePool resource) {
     public void delete(KafkaNodePool resource) {
         String namespaceName = resource.getMetadata().getNamespace();
         String clusterName = resource.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL);
+        String podsetName = String.join("-", clusterName, resource.getMetadata().getName());
 
         kafkaNodePoolClient().inNamespace(resource.getMetadata().getNamespace()).resource(resource).delete();
 
         // once KafkaNodePool is deleted, we should delete PVCs and wait for the deletion to be completed
         // the Kafka pods will not be deleted during Kafka resource deletion -> so if we would delete PVCs during Kafka deletion
         // they will not be deleted too
         // additional deletion of pvcs with specification deleteClaim set to false which were not deleted prior this method
-        PersistentVolumeClaimUtils.deletePvcsByPrefixWithWait(namespaceName, clusterName);
+        PersistentVolumeClaimUtils.deletePvcsByPrefixWithWait(namespaceName, podsetName);
```

Comment:
This is just fix that is needed in order to clear all resources after the test finishes